components/movementsensor/replay/replay_test.go

```diff
@@ -78,25 +78,25 @@ var (
 		{OX: 2, OY: 1, OZ: 1, Theta: 0},
 	}
 
-	defaultMaxDataLength = map[string]int{
-		"Position":           len(positionPointData),
-		"LinearAcceleration": len(linearAccelerationData),
-		"AngularVelocity":    len(angularVelocityData),
-		"LinearVelocity":     len(linearVelocityData),
-		"Orientation":        len(orientationData),
-		"CompassHeading":     len(compassHeadingData),
+	allMethodsMaxDataLength = map[method]int{
```

Comment:
Renamed this and the one below to match the variable names I'm using in the features PR (such as `allMethodsSupported` and `noMethodsSupported`)